web/src/SidebarTriggerButton.test.tsx

```diff
@@ -1,7 +1,32 @@
 import React from "react"
 import { mount } from "enzyme"
-import SidebarTriggerButton from "./SidebarTriggerButton"
-import { TriggerMode } from "./types"
+import SidebarTriggerButton, {
+  TriggerButtonTooltip,
+} from "./SidebarTriggerButton"
+import { Resource, ResourceView, TriggerMode } from "./types"
+import { oneResource, twoResourceView } from "./testdata"
+import Sidebar, { SidebarItem } from "./Sidebar"
+import { MemoryRouter } from "react-router"
+import PathBuilder from "./PathBuilder"
+
+let pathBuilder = new PathBuilder("localhost", "/")
+
+let assertTriggerButtonProps = (
+  button: any,
+  expectClickable: boolean,
```

Comment:
this seems hard to maintain...it might be easier to pass in a list of classes. @hyu what do you think?

Reply:
i guess a more javascript-y way to do this would be to have a type like 
{isCickable?: boolean, isSelected?: boolean} so that the caller only has to pass the ones that are true, tho @hyu might have better ideas